### DISCUSSION TITLE
younger

### DISCUSSION THREAD

**Discussion by chgvgiftutcnbcyz** [community/community#48967] [Programming Help]
### Select Topic Area

Question

### Body

how can I write on python using functions

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @chgvgiftutcnbcyz ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
